### PR TITLE
doc: Improve English description

### PR DESCRIPTION
**Pull request by JHH20**
Correct grammatical errors and re-write some sentences to make it easier to understand.

### CODE REVIEW

**Comment by kangtegong** [comment]
Hello, I think squashing your commit into one whould be more nicer. Can you sqash your commits?

**Comment by JHH20** [comment]
Ok, I have squashed them into a single commit.

File: INSTALL.md [hunk 1]
```diff
@@ -45,29 +45,29 @@ On redhat based systems (like Fedora, RHEL), it'll be `elfutils-devel`.
 
     $ sudo dnf install elfutils-devel
 
-It also uses libstdc++ library to demangle C++ symbols in full detail.
+The libdw library also uses libstdc++ library to demangle C++ symbols in full detail.
```
Comment (honggyukim) [inline INSTALL.md:48]: The context might be a bit confusing but `It` means `uftrace` here. So I would suggest this.
```suggestion
uftrace also uses libstdc++ library to demangle C++ symbols in full detail.
```

File: INSTALL.md [hunk 2]
```diff
@@ -91,12 +91,12 @@ Then you can run `make` to build the source.
 
     $ make
 
-It builds uftrace and resulting binary exists in the current directory.
+It builds uftrace, placing the resulting binary in the current directory.
```
Comment (honggyukim) [inline INSTALL.md:94]: `make` produces `uftrace` and `libmcount*.so` shared objects so better to be plural here.
```suggestion
It builds uftrace, placing the resulting binaries in the current directory.
```

File: INSTALL.md [hunk 3]
```diff
@@ -105,9 +105,9 @@ build command lines with V=1 (like kernel).
 CONFIGURATION
 =============
 
-The uftrace implements own version of configure script to save user
-preferences.  The config file (named `.config`) is created if not exist
-on build time with default options.  User can set custom installation
+The uftrace implements its own version of configure script to save user
+preferences.  The config file (named `.config`) is created, if non existent,
```
Comment (honggyukim) [inline INSTALL.md:109]: I'm not sure if "[If non existent](https://www.google.com/search?q=%22if+non+existent%22&oq=%22if+non+existent%22&aqs=chrome..69i57.1046j0j1&sourceid=chrome&ie=UTF-8)" is more common than "[if not exist](https://www.google.com/search?q=%22if+not+exist%22&oq=%22if+not+exist%22&aqs=chrome..69i57.4296j0j1&sourceid=chrome&ie=UTF-8)".

Google search result show "if not exist" is much more common.

File: doc/uftrace-dump.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ DUMP OPTIONS
 :   Show hex dump of data as well
 
 \--sample-time=*TIME*
-:   Apply sampling time when generating output for --flame-graph.  By default it
+:   Apply sampling time when generating output for --flame-graph.  By default, this command
```
Comment (honggyukim) [inline doc/uftrace-dump.md:38]: Not sure if we have to change this way. If "it" is not suitable here then "this" might be shorter and enough.

**Review by honggyukim — COMMENTED** [review]
Superb! You've read the pages really carefully. I've put some comments but mostly looks good to me.

File: INSTALL.md (hunk 3 above)
Comment (JHH20) [reply INSTALL.md:109]: > I'm not sure if "[If non existent](https://www.google.com/search?q=%22if+non+existent%22&oq=%22if+non+existent%22&aqs=chrome..69i57.1046j0j1&sourceid=chrome&ie=UTF-8)" is more common than "[if not exist](https://www.google.com/search?q=%22if+not+exist%22&oq=%22if+not+exist%22&aqs=chrome..69i57.4296j0j1&sourceid=chrome&ie=UTF-8)".
> 
> Google search result show "if not exist" is much more common.

`if not exist` is closer to code syntax (if !exists) than English (if [it does] not exist) so for documentation, I think it is better to change it.

If `non existent` feels awkward, how about `if missing` or `if not found` ?

Comment (honggyukim) [reply INSTALL.md:109]: Then let's make it as a proper English sentence so I think `If it doesn't exist` is better.

**Comment by honggyukim** [comment]
Please update the PR based on the review comments. Thanks!

**Review by honggyukim — APPROVED** [review]
Thanks for the update. LGTM from myside. @namhyung will have a final look.

**Review by namhyung — APPROVED** [review]
LGTM, thanks for fixing my poor English! :)

**Pull request merged**